assets/l10n/app_en.arb

```diff
@@ -348,12 +348,18 @@
   "@composeBoxSelfDmContentHint": {
     "description": "Hint text for content input when sending a message to yourself."
   },
-  "composeBoxChannelContentHint": "Message #{channel} > {topic}",
+  "composeBoxChannelContentHint": "Message #{channel}",
```

Comment:
The reasoning in the last commit:

    compose: Avoid translating Zulip message destinations
    
    The '#channel > topic' style strings are not supposed to be translated
    into different languages as they are Zulip's language of expressing the
    desintation, not something bound to the English language.
    
    See also:
      https://github.com/zulip/zulip-flutter/pull/1148#discussion_r1941990585

applies equally well to this string, right?

So maybe let's keep this as a single translated string, and just pass either "#general" or "#general > lunch" to it from app code.